src/components/AppointmentForm/formikControlsCfg.js

```diff
@@ -0,0 +1,20 @@
+const formikControls = [
```

Comment:
short name for file/const could just be `formConfig` or more specific `appointmentsFormConfig`, and it's better to have full name, even for obvious cfg = config - don't worry it's long, it's more readable